parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.kt

```diff
@@ -36,12 +36,14 @@ public interface ParselyTracker {
      *                    content). Do not use this for **content also hosted on** URLs Parse.ly
      *                    would normally crawl.
      * @param extraData   A Map of additional information to send with the event.
+     * @param siteIdSource The source of the site ID to use for the event.
```

Comment:
Perhaps it is worth it to add / repeat / emphasize a bit more info on siteId and what's the default behavior?